src/cmd-run

```diff
@@ -193,6 +193,22 @@ EOF
 rowcol=$(stty -a | tr ';' '\n' | grep -e 'rows\|columns' | tr '\n' ' ' )
 rowcol=$(echo "stty ${rowcol}" | base64 --wrap 0)
 
+# if SSH port mapping is requested, automagically inject all the pubkeys
+ssh_pubkeys=""
```

Comment:
This should use the ssh agent, not look for pubkeys. This notably breaks my gpg-derived-on-yubikey setup

Reply:
Yeah, my SSH keys are only in the agent too.

Reply:
OK I think I got this correct.